config/default.json

```diff
@@ -26,15 +26,15 @@
         "persistDuplicateCache": false
     },
     "logger": {
-        "consoleLogLevel": "verbose",
+        "consoleLogLevel": "warn",
```

Comment:
Think default for this should be verbose. The point is you can override this

Reply:
Sure , one can, but most people will stick to the defaults until they run into problems.
And what's the benefit of writing everything twice in verbose? console log will most of the time be caught by journald/syslog and there it's rather not custom to put debug logs in.

And if someone really starts manually an looks at the console it's anyway just for debugging purpose but not normal run.